src/common.ts

```diff
@@ -23,7 +23,7 @@ export const endpointMap = {
   getUser: `${BASE_URL}/v2/user`,
   getUserEvents: `${BASE_URL}/v3/events`,
   deleteUser: `${BASE_URL}/v1/user`,
-  listDeployment: `${BASE_URL}/v6/deployments`,
+  deploymentList: `${BASE_URL}/v6/deployments`,
```

Comment:
I think we should rename this to `listDeployments`. That's pretty much the convention we're following. {verb}{Resource}. eg `getUser`, `getUserEvents`, `listAliases` etc.